### PR TITLE
READY: Add org.tomviz.tomviz

### PR DESCRIPTION
**Pull request by kevinsmia1939**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/OpenChemistry/tomviz/issues/2157
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
Can anyone explain why I'm getting errror downloading?

**Comment by barthalion** [comment]
Seems to be upstream issue to me?

```
Error downloading object: data/README.md (b30a14a): Smudge error: Error downloading data/README.md (b30a14a308f64c6fc2969e2b959d79dacdc5affda1d1c0e24f8e176304147146): Error transferring "b30a14a308f64c6fc2969e2b959d79dacdc5affda1d1c0e24f8e176304147146": [0] remote missing object b30a14a308f64c6fc2969e2b959d79dacdc5affda1d1c0e24f8e176304147146
```

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "shared-modules"]
+	path = shared-modules
+	url = git://github.com/flathub/shared-modules.git
```
Comment (barthalion) [inline .gitmodules:3]:
```suggestion
	url = https://github.com/flathub/shared-modules.git
```

File: org.tomviz.Tomviz.yaml [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+      - -DPARAVIEW_ENABLE_EMBEDDED_DOCUMENTATION:BOOL=OFF
+      - -DPARAVIEW_USE_QTHELP:BOOL=OFF
+      - -DPARAVIEW_PLUGINS_DEFAULT:BOOL=OFF
+      - -DVTK_SMP_IMPLEMENTATION_TYPE:STRING=TBB
+      - -DVTK_PYTHON_VERSION:STRING=3
+      - -DVTK_PYTHON_FULL_THREADSAFE:BOOL=ON
+      - -DVTK_NO_PYTHON_THREADS:BOOL=OFF
+    sources:
+      - type: git
+        url: git://github.com/OpenChemistry/ParaView.git
```
Comment (barthalion) [inline org.tomviz.Tomviz.yaml:164]:
```suggestion
        url: https://github.com/OpenChemistry/ParaView.git
```

**Comment by barthalion** [comment]
bot, build org.tomviz.Tomviz

**Comment by barthalion** [comment]
Although yeah, it cloned fine locally. I'll poke around.

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
It still cannot download file for some reason, it work properly locally.

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
Why does it fail to reach the image? The link is still online.

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Comment by kevinsmia1939** [comment]
bot, build org.tomviz.Tomviz

**Pull request closed without merging**